app/views/posts/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Index posts</h1>
+<% @posts.each do | post|%>
+ <div class="border border-dark bg-white">
```

Comment:
Please create a method and use it to render the 5 most recent comments in the `app/views/posts/index.html.erb`. There was an error when I tried to access the posts index page. Please make this page work and follow [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png) wireframe for all posts page.
![error](https://user-images.githubusercontent.com/35504149/151371277-46f34d80-98e0-460f-9a16-d4c08a973f44.png)